ext/skeleton/config.m4.in

```diff
@@ -62,19 +62,19 @@ if test "$PHP_%EXTNAMECAPS%" != "no"; then
   dnl LIBNAME=%EXTNAMECAPS% # you may want to change this
   dnl LIBSYMBOL=%EXTNAMECAPS% # you most likely want to change this
 
-  dnl If you need to check for a particular library function
-  dnl and you are using pkg-config:
+  dnl If you need to check for a particular library function (e.g. a conditional
+  dnl or version-dependent feature) and you are using pkg-config:
   dnl PHP_CHECK_LIBRARY($LIBNAME, $LIBSYMBOL,
   dnl [
-  dnl   PHP_EVAL_INCLINE($LIBFOO_CFLAGS)
-  dnl   PHP_EVAL_LIBLINE($LIBFOO_LIBS, %EXTNAMECAPS%_SHARED_LIBADD)
   dnl   AC_DEFINE(HAVE_%EXTNAMECAPS%LIB, 1, [ ])
   dnl ],[
   dnl   AC_MSG_ERROR([wrong %EXTNAME% lib version or lib not found])
```

Comment:
I think we should also adjust the define and this message. Something like `HAVE_%EXTNAMECAPS%_FEATURE` and `%EXTNAME% lib does not support feature`.

Reply:
@nikic, good idea. I think `HAVE_%EXTNAMECAPS%_FEATURE` is fine. What do you think about `FEATURE not supported by your %EXTNAME% library.` for the error message?

Reply:
Sounds good.

Reply:
Thanks, @nikic. I've updated the PR.